app/src/renderer/vuex/modules/delegates.js

```diff
@@ -16,7 +16,12 @@ export default ({ node }) => {
       state.loading = loading
     },
     setDelegates(state, delegates) {
-      state.delegates = delegates
+      state.delegates = delegates.map(delegate => {
+        delegate.id = delegate.owner
+        delegate.voting_power = ratToBigNumber(delegate.tokens)
+
+        return delegate
+      })
```

Comment:
It's better not to mutate an input if you can avoid it:

```JavaScript
state.delegates = delegates.map(delegate =>
  Object.assign({}, delegate, {
    id: delegate.owner,
    voting_power: ratToBigNumber(delegate.tokens)
  })
)
```

Reply:
Vue doesn't deep watch arrays. If I use this pattern, Vue doesn't know the validators changed in PageValidator. I don't want to handle this issue in this PR.

Reply:
OK.  In the future it would be better then to use `forEach` to signal that you're deliberately making side effects:

```JavaScript
delegates.forEach(delegate => {
  delegate.id = delegate.owner
  delegate.voting_power = ratToBigNumber(delegate.tokens)
})

state.delegates = delegates
```

Reply:
I completely agree. My mistake.